src/system_time.rs

```diff
@@ -0,0 +1,27 @@
+use std::time::Duration;
```

Comment:
Should this import be moved into the `now()` function on line 16?

Reply:
It is also used in the return type for `duration_since_unix_epoch`